src/Sylius/Bundle/ChannelBundle/Tests/DependencyInjection/Compiler/CompositeRequestResolverPassTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ChannelBundle\Tests\DependencyInjection\Compiler;
+
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractCompilerPassTestCase;
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\DefinitionHasMethodCallConstraint;
+use Sylius\Bundle\ChannelBundle\DependencyInjection\Compiler\CompositeRequestResolverPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class CompositeRequestResolverPassTest extends AbstractCompilerPassTestCase
+{
+    /**
+     * @test
+     */
+    public function it_collects_tagged_request_based_channel_contexts()
```

Comment:
I did snake_case in #3936 and I think we should use it because it is more readable. That being said, I am not sure how that will work out for API tests in PHPUnit:

```
function it_allows_creating_product()
function it_denies_access_to_product_creation_for_not_authenticated_users()
function it_returns_not_found_response_for_non_existing_product()
function it_creates_customer_without_user_account()
function it_allows_to_delete_users()
function it_allows_to_update_users()
```

Thoughts?